docs/user-guide/commands.md

```diff
@@ -79,7 +79,7 @@ usage: |
 maintainer: foo@bar.com # Maintainer of the command
 format: binary # Format the command is in (binary, habitat)
 binary:
-    file: ./foobar.sh # Path to script or binary file from repository root
+    file: ./foobar.sh # Path to script or binary file from sd-command.yaml file
```

Comment:
`relative path to script or binary file ...`

Reply:
I fixed it. Thanks.